### PR TITLE
Add reload button to overlay after a while

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

The app-backend sometimes get stuck on too much load. This is not ideally and it is something we're looking into fixing.
In the mean time, this quick fix would let users know that things are taking too long and suggest that they reload the page to force a new request.

### Preview

Screenshot 2024-01-04 at 14 10 10

### Double check

- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: web/src/components/LoadingOverlay.tsx [hunk 1]
```diff
@@ -2,25 +2,62 @@ import { animated, useTransition } from '@react-spring/web';
...
+            {showButton && (
+              <>
+                <p>{__('misc.slow-loading-text')}</p>
+                <Button
+                  className="w-20 min-w-min dark:bg-gray-800/80"
+                  aria-label="Reload page"
+                  onClick={() => window.location.reload()}
+                >
+                  {__('misc.reload')}
+                </Button>
```
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/src/components/LoadingOverlay.tsx:58]: Eh, not worth it for this use case.

[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/src/components/LoadingOverlay.tsx:58]: No thank you bot, we only use classNames for tailwind styling purposes :)

File: web/src/components/LoadingOverlay.tsx [hunk 2]
```diff
@@ -2,25 +2,62 @@ import { animated, useTransition } from '@react-spring/web';
...
+              </>
+            )}
```
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/src/components/LoadingOverlay.tsx:60]: I actually disagree on readability here, I think the ternary usage for conditional rendering gets very messy and hard to follow. Using `&&` makes it clear that there is no "else" part.

**Comment by VIKTORVAV99** [comment]
Will review this after lunch :)

File: web/src/components/LoadingOverlay.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply web/src/components/LoadingOverlay.tsx:60]: Yeah, I agree.

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM! 🎉

**Comment by VIKTORVAV99** [comment]
@madsnedergaard can we merge this? I want to get rid of some open PRs 😅
